src/administration/web/configure-environment.md

```diff
@@ -15,7 +15,7 @@ environment.
 The *Delete* action will destroy all services running on this environment (PHP, MySQL, volumes...)
 so that only the Git branch remains. You can reactivate the environment later if needed.
 
-To also delete the branch, you need to use Git and run:
+To delete the branch also, you need to execute the following:
```

Comment:
With also in that position, "as well" works better.  Or leave the also where it is.  Either one is legit I think.